tests/Microsoft.Data.DataFrame.Tests/Microsoft.DataFrames.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Similarly here, should it be either `Microsoft.Data.Tests` or `Microsoft.DataFrame.Tests`? If we want it to be `Microsoft.Data.DataFrame`, then let's be consistent in the source project/namespace as well. I don't have a strong opinion one way or the other.